charts/kong/values.yaml

```diff
@@ -647,7 +647,8 @@ ingressController:
 
   konnect:
     enabled: false
-
+    # Deprecated: Specifies a Konnect Runtime Group's ID that the controller will push its data-plane config to.
```

Comment:
Should we remove the `runtimeGroupID` in the default values?

Reply:
Good point. I left it here so that some people can check the helm chart after getting the configuration from Konnect.

WDYT?